daemon/src/lib.rs

```diff
@@ -346,3 +349,43 @@ where
             .await?
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub enum Environment {
+    UMBREL,
+    RASPIBLITZ,
+    DOCKER,
+    BINARY,
```

Comment:
would you be open to changing `BINARY` to `DESKTOP_APP`or `EXECUTABLE`  or something? it might be my personal awkwardness, but I never refer to the apps I run as binaries. I get that this might be subjective, so please feel free to ignore this comment :)